core/move.js

```diff
@@ -40,8 +41,8 @@ function move(side /*: SideName */, src /*: Metadata */, dst /*: Metadata */) {
   if (!dst.overwrite) {
     delete dst._rev
   }
-  delete dst.sides
-  metadata.markSide(side, dst)
+  reinitializeSides(dst, src)
```

Comment:
I would have expected this to break stuff, I'm surprised it doesn't...

Reply:
Not sure why it is needed either... Wouldn't having `{[side]: 1}` already ensure the addition to the destination is synced?

Reply:
Tests are indeed passing without this commit. I removed it and will keep it in another branch just in case I figure out a situation where it's necessary.

Reply:
I figured out why it's needed :).

For successive child moves. If we don't keep both existing sides in child moves and later want to move those children, we'll end up detecting only one side on them and convert those moves to additions later resulting in sync errors because their remote revisions would not have been updated.